content/concepts/the-rippled-server/transaction-censorship-detection.md

```diff
@@ -0,0 +1,42 @@
+# Transaction Censorship Detection
+
+The XRP Ledger is designed to be [censorship resistant](xrp-ledger-overview.html#censorship-resistant-transaction-processing). Any attempt to censor transactions requires coordinated action by a majority of the XRP Ledger network's validators.
```

Comment:
I don't think it requires coordination by a majority of validators to _attempt_ to censor transactions. I mean, whether they succeed is a different matter. 😉

Especially if it's just delaying transactions to later ledger versions (which could be relevant when it comes to chasing high-value offers in the decentralized exchange, aka front-running), I would think that temporarily censoring certain transactions—just delaying them a ledger version or two—might be possible with a relatively-small number of validators. If transactions that are on the "avalanche" threshold of being included in a consensus round or not, it seems like a few validators could be the deciding vote causing those transactions to be delayed to the next ledger version.

I don't know whether this censorship detector would recognize that sort of behavior, but it is a malicious abuse of consensus that seems possible.